### PR TITLE
[Workers] Update "Custom Ports" known issue to reflect dev-time only behaviour

### PR DESCRIPTION
**Pull request by xdjoshuaaz**
The change made in PR #7514 assumes that custom ports work with HTTP requests, however, this is not true.

PR #5255 did correctly mention the dev-time discrepancy but it was not merged.

Weirdly, it is possible to open up a TCP socket to the HTTP server on the custom port using the recent [TCP sockets runtime API](https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/), and craft a HTTP request + parse the HTTP response by hand.

Because of this, the decision to restrict custom ports when using `fetch` may be now be defeated by this new TCP socket API?

### CODE REVIEW

File: content/workers/platform/known-issues.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ return await fetch(request);
 
 ## Custom ports
 
-For Workers subrequests, custom ports are ignored when using HTTPS and are instead always sent to port `443`. When using HTTP, custom ports are respected.
+For Workers subrequests, custom ports are ignored and requests are sent to the scheme's default port, such as `443` for HTTPS.
```
Comment (irvinebroque) [inline content/workers/platform/known-issues.md]:
```suggestion
For Workers subrequests, when a Worker is deployed, custom ports are ignored and requests are sent to the scheme's default port, such as `443` for HTTPS. Note that when developing a Worker locally, or from within the Cloudflare dashboard using Quick Edit, custom ports are respected and allowed.
```

File: content/workers/platform/known-issues.md [hunk 2]
```diff
@@ -45,7 +45,9 @@ return await fetch(request);
...
+
+This behaviour is not present in local development or when using Quick Edit in the dashboard.
```
Comment (irvinebroque) [inline content/workers/platform/known-issues.md]:
```suggestion
```

File: content/workers/platform/known-issues.md [hunk 3]
```diff
@@ -45,7 +45,9 @@ return await fetch(request);
 
 ## Custom ports
 
-For Workers subrequests, custom ports are ignored when using HTTPS and are instead always sent to port `443`. When using HTTP, custom ports are respected.
+For Workers subrequests, custom ports are ignored and requests are sent to the scheme's default port, such as `443` for HTTPS.
+
```
Comment (irvinebroque) [inline content/workers/platform/known-issues.md]:
```suggestion
```

**Review by irvinebroque — APPROVED** [review]
Thanks for flagging this. Small copy edits but 🚀 

@mrbbot @admah @lrapoport-cf for visibility — wonder if we should update local dev / Quick Editor to more closely mirror production behavior.

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Comment by admah** [comment]
+1 to the comment from @mrbbot. As long as it's documented here, I'm good with leaving the local dev experience as-is.

**Pull request merged**